### PR TITLE
normalize data explorer code

### PR DESCRIPTION
**Pull request by zarsky-broad**
This started with wanting to remove the warning that appears when installing deps:
```
npm WARN react-iframe-resizer-super@0.2.2 requires a peer of iframe-resizer@^3.6.2 but none is installed. You must install peer dependencies yourself.
```

After some searching, I decided that the best way to do this was to remove `react-iframe-resizer-super` entirely, as it hasn't been updated recently, in favor of the library it wraps, since we don't use any advanced features.

While making the changes to do this, I made a few tweaks to file naming and location to bring the style of this code more in line with what we normally create.

### CODE REVIEW

**Review by melissachang — APPROVED** [review]
LG. I tested with the manual steps in #1644

File: src/components/DataExplorerFrame.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+          })
+        } else if (deQueryStr) {
+          // Propagate Data Explorer URL params to app.terra.bio.
+          // Don't call Nav.history.replace(). That will trigger a request and
+          // cause the page to flicker.
+          window.history.replaceState({}, `Data Explorer - ${dataset}`, `#${Nav.history.location.pathname.slice(1)}?${deQueryStr}`)
+        }
+      }
+    }, elem.current)
+  }, [dataset])
```
Comment (panentheos) [inline src/components/DataExplorerFrame.js:31]: This should probably return a cleanup handler that calls `removeListeners`.

File: src/components/DataExplorerFrame.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+import { h, iframe } from 'react-hyperscript-helpers'
+import datasets from 'src/data/datasets'
+import * as Nav from 'src/libs/nav'
+
+
+const DataExplorerFrame = ({ dataset }) => {
+  const elem = useRef()
+  const { origin } = _.find({ name: dataset }, datasets)
+
+  useLayoutEffect(() => {
```
Comment (panentheos) [inline src/components/DataExplorerFrame.js]: Does this need to be `useLayoutEffect`? The docs recommend using `useEffect` if possible.

**Review by panentheos — APPROVED** [review]
Looks good, see comments

**Pull request merged**